src/Bridge/Symfony/Bundle/Resources/views/Redoc/index.html.twig

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>{% if title %}{{ title }} - {% endif %}API Platform</title>
+    <meta charset="utf-8"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://fonts.googleapis.com/css?family=Montserrat:300,400,700|Roboto:300,400,700" rel="stylesheet">
+    <style>
+        body {
+            margin: 0;
+            padding: 0;
+        }
+    </style>
+</head>
+<body>
+    <redoc spec-url='{{ redoc_data['url'] }}'></redoc>
```

Comment:
As done in Swagger UI, it would be better to use the `spec` prop instead of `spec-url` to prevent a useless AJAX request.

IMO it's another reason to merge ReDoc and Swagger UI support in the same template and controller.